README.md

```diff
@@ -121,6 +121,20 @@ The specifications that we are using at Prolific Interactive are this one:
 }
 ```
 
+* apiInactive
+    * Determines if the API is active or not. If it is `true` the App UI should be blocked until this is `false`.
+    * required, type: boolean
+* forceUpdate
+    * Determines if the App should force the user to update to the latest version. This should be used very sparingly.
+    Only in emergencies or non supported app versions.
+    * required, type: boolean
+* retryInterval
+    * The time in seconds to wait to ping this endpoint until `apiInactive` is set to `false`.
+    * required, type: integer
```

Comment:
this doesn't seem to match up with what's in the example where `retryInterval` value is null